support/k8s/k8s-workload-registrar/mode-crd/webhook/webhook_svid.go

```diff
@@ -91,7 +92,13 @@ func (e *SVID) MintSVID(ctx context.Context, key crypto.Signer) (err error) {
 
 	// Mint new SVID
 	var resp *svidv1.MintX509SVIDResponse
-	err = retry.OnError(retry.DefaultBackoff, e.mintSVIDRetry, func() (err error) {
+	backoff := wait.Backoff{
+		Steps:    11,
+		Duration: 15 * time.Millisecond,
+		Factor:   2.0,
+		Jitter:   0.1,
+	}
+	err = retry.OnError(backoff, e.mintSVIDRetry, func() (err error) {
```

Comment:
How long was the backoff for previous to this change? One minute feels like a long time to detect a misconfiguration...

Reply:
about 1.5 seconds. On slack someone mentioned needing up to a minute because they had a log of OPA rules to process, so thats where the 1 minute number comes from. It is a bit long, i agree.